client/state/jetpack-agency-dashboard/selectors.ts

```diff
@@ -16,6 +16,9 @@ export const JETPACK_DASHBOARD_WELCOME_BANNER_PREFERENCE_HOME_PAGE =
 export const JETPACK_DASHBOARD_SURVEY_BANNER_PREFERENCE =
 	'jetpack-dashboard-agency-program-survey-banner-preference';
 
+export const JETPACK_DASHBOARD_CHECKOUT_REDIRECT_MODAL_DISMISSED =
+	'jetpack-dashboard-checkout-redirect-modal-dismissed';
```

Comment:
Not a blocker.

Maybe we can add the word `agency-program` in the preference name just to make it more specific to the agency program and not overlap with the Jetpack dashboard in the core plugin. wdyt?

Reply:
Doesn't really matter. But, I have changed this.